srcpkgs/Waybar/template

```diff
@@ -1,13 +1,14 @@
 # Template file for 'Waybar'
 pkgname=Waybar
-version=0.8.0
-revision=2
+version=0.9.0
+revision=1
 build_style=meson
 configure_args="-Dlibnl=$(vopt_if libnl enabled disabled)
  -Dpulseaudio=$(vopt_if pulseaudio enabled disabled)
- -Ddbusmenu-gtk=$(vopt_if dbusmenugtk enabled disabled)"
-hostmakedepends="pkg-config glib-devel wayland-devel sway"
-makedepends="libinput-devel wayland-devel wlroots-devel gtkmm-devel spdlog
+ -Ddbusmenu-gtk=$(vopt_if dbusmenugtk enabled disabled)
+ -Dsystemd=disabled"
+hostmakedepends="pkg-config glib-devel wayland-devel sway" 
+makedepends="gtk-layer-shell scdoc libinput-devel wayland-devel wlroots-devel gtkmm-devel spdlog
```

Comment:
`gtk-layer-shell` isn't a package, as of now. It seems that the only package that uses it brings gtk-layer-shell's sources into it's tree. See `wf-shell`.